src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,313 @@
+
+In this tutorial, you will connect to:
+
+- The _system interface_ for administration tasks in both clusters, and starting the replication stream from the standby cluster.
+- The application _virtual cluster_ on the primary to work with databases, tables, workloads, and so on.
+
+{% include {{ page.version.version }}/physical-replication/reference-links-replication.md %}
+
+## Overview
+
+The high-level steps in this tutorial are:
+
+1. Create and start the primary cluster.
+1. Configure and create a user on the primary cluster.
+1. Create and start the standby cluster.
+1. Configure and create a user on the standby cluster.
+1. Verify certificates.
+1. Start the replication stream from the standby cluster.
+
+## Before you begin
+
+{% include enterprise-feature.md %}
+
+- Two separate CockroachDB clusters (primary and standby) with a minimum of three nodes each using the same CockroachDB {{page.version.version}} version.
+    - To set up each cluster, you can follow the [Deploy CockroachDB on Premises]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}). When you start each node in your cluster with the `cockroach start` command, you **must** pass the `--config-profile` flag with a `replication` value. Refer to cluster creation steps for the [primary](#start-the-primary-cluster) and for the [standby](#start-the-standby-cluster) cluster to do this.
+- All nodes in each cluster will need access to the Certificate Authority for the other cluster. Refer to [Verify certificates](#step-3-verify-certificates).
+- An [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/enterprise-licensing.md %}) on the primary **and** standby clusters. You must use the system interface on the primary and standby to enable your {{ site.data.products.enterprise }} license.
+
+## Step 1. Create the primary cluster
+
+### Start the primary cluster
+
+To enable physical cluster replication, it is necessary to start each node with the appropriate _configuration profile_ set with the `--config-profile` flag. A configuration profile applies a custom configuration to the server at initialization time. When using physical cluster replication, the `replication-source` and `replication-target` configuration profiles are used to create a virtualized cluster with a system interface and an application virtual cluster.
+
+The primary cluster requires the following value:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--config-profile replication-source
+~~~
+
+For example, a `cockroach start` command according to the [prerequisite deployment guide]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}#step-3-start-nodes):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach start \
+--certs-dir=certs \
+--advertise-addr=<node1 address> \
+--join=<node1 address>,<node2 address>,<node3 address> \
+--cache=.25 \
+--max-sql-memory=.25 \
+--background \
```

Comment:
```suggestion
```
Don't start nodes in the background. it's better to start in the foreground so they can see what's happening. See https://github.com/cockroachdb/docs/pull/18033 and [DOC-523] for context.